ryoanji/src/ryoanji/nbody/traversal_cpu.hpp

```diff
@@ -159,12 +153,10 @@ void computeGravityGroup(const util::array<Vec4<T1>, N>& target, const TreeNodeI
  *
  * only computes total gravitational energy, no potential per particle
  *
- * @tparam KeyType               unsigned 32- or 64-bit integer type
- * @tparam T1                    float or double
- * @tparam T2                    float or double
+ * @tparam       MType           Multipole type including expansion order, e.g. spherical or cartesian
  * @param[in]    childOffsets    child node index of each node
  * @param[in]    internalToLeaf  map to convert an octree node index into a cstone leaf index
- * @param[in]    numLeafNodes    number of leaf nodes
+ * @param[in]    centers         (x,y,z,mac^2) expansion center for each tree cell
```

Comment:
If there is going to be more information in this array that just centers, perhaps this is describing an "cellSphere" or "interactionSphere" or "openingSphere".

Reply:
Maybe macSphere would be even more to the point?